### PR TITLE
Add Kleros verification process via verify tab

### PR DESCRIPTION
**Pull request by pimato**

### CODE REVIEW

**Comment by liaojikunwork** [comment]
Hi, please resolve the conflicts.

**Comment by KadenZipfel** [comment]
This market seems to be verified, but displaying 'Not Verified': http://localhost:3000/#/0xd47616521b09caa98eff7707d9c3358a67699b69

image

File: app/.env.example [hunk 1]
```diff
@@ -0,0 +1,58 @@
+#############
```
Comment (KadenZipfel) [inline app/.env.example:1]: If we want to go ahead with the removal of the `env` file, we should remove this file

File: app/src/common/constants.ts [hunk 2]
```diff
@@ -56,6 +56,18 @@ export const GRAPH_MAINNET_WS = 'wss://api.thegraph.com/subgraphs/name/protofire
...
+export const KLEROS_CURATE_GRAPH_MAINNET_WS =
+  process.env.REACT_APP_KLEROS_CURATE_GRAPH_MAINNET_WS || 'wss://api.thegraph.com/subgraphs/name/kleros/curate'
+export const KLEROS_CURATE_GRAPH_RINKEBY_HTTP =
+  process.env.REACT_APP_KLEROS_CURATE_GRAPH_RINKEBY_HTTP ||
+  'https://api.thegraph.com/subgraphs/name/kleros/curate-rinkeby'
+export const KLEROS_CURATE_GRAPH_RINKEBY_WS =
+  process.env.REACT_APP_KLEROS_CURATE_GRAPH_RINKEBY_WS || 'wss://api.thegraph.com/subgraphs/name/kleros/curate-rinkeby'
+
+export const IPFS_GATEWAY = process.env.REACT_APP_IPFS_GATEWAY || 'https://ipfs.kleros.io'
+
```
Comment (KadenZipfel) [inline app/src/common/constants.ts:70]: We should also remove the `env` variables here if we're removing the `env` file

**Review by KadenZipfel — COMMENTED** [review]
Other than `env` considerations it looks good to me!

**Review by Mi-Lan — COMMENTED** [review]
@hexyls So it seems that this is some leftover from env in serviceWorker.js file?
or not?
Screenshot 2020-11-26 at 21 33 54

**Review by hexyls — APPROVED** [review]
Huge effort! Well done everyone 🎉

**Review by Mi-Lan — APPROVED** [review]
Totally! Good job!

**Pull request merged**